google/cloud/bigquery/table.py

```diff
@@ -2077,7 +2077,7 @@ def to_geodataframe(
                   Use the :func:`tqdm.tqdm` function to print a progress bar
                   to :data:`sys.stderr`.
                 ``'tqdm_notebook'``
```

Comment:
does this line also need to change? (it might not but I just noticed it stayed the same)

Reply:
Yep, it should have gotten changed with the function changed, but I'll update it for this PR!